content/docs/1.6.0/references/helm-values.md

```diff
@@ -5,175 +5,173 @@ weight: 5
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| longhornUI.nodeSelector | object | `{}` | Select nodes to run Longhorn UI |
-| longhornUI.priorityClass | string | `nil` | Priority class count for longhorn ui |
-| longhornUI.replicas | int | `2` | Replica count for longhorn ui |
-| longhornUI.tolerations | list | `[]` | Tolerate nodes to run Longhorn UI |
+| longhornUI.nodeSelector | object | `{}` | Node selector for Longhorn UI. Specify the nodes allowed to run Longhorn UI. |
+| longhornUI.priorityClass | string | `nil` | PriorityClass for Longhorn UI. |
+| longhornUI.replicas | int | `2` | Replica count for Longhorn UI. |
+| longhornUI.tolerations | list | `[]` | Toleration for Longhorn UI on nodes allowed to run Longhorn components. |
 
 ### Ingress Settings
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| ingress.annotations | string | `nil` | Ingress annotations done as key:value pairs |
-| ingress.enabled | bool | `false` | Set to true to enable ingress record generation |
-| ingress.host | string | `"sslip.io"` | Layer 7 Load Balancer hostname |
-| ingress.ingressClassName | string | `nil` | Add ingressClassName to the Ingress Can replace the kubernetes.io/ingress.class annotation on v1.18+ |
-| ingress.path | string | `"/"` | If ingress is enabled you can set the default ingress path then you can access the UI by using the following full path {{host}}+{{path}} |
-| ingress.secrets | string | `nil` | If you're providing your own certificates, please use this to add the certificates as secrets |
-| ingress.secureBackends | bool | `false` | Enable this in order to enable that the backend service will be connected at port 443 |
-| ingress.tls | bool | `false` | Set this to true in order to enable TLS on the ingress record |
-| ingress.tlsSecret | string | `"longhorn.local-tls"` | If TLS is set to true, you must declare what secret will store the key/certificate for TLS |
+| ingress.annotations | string | `nil` | Ingress annotations in the form of key-value pairs. |
+| ingress.enabled | bool | `false` | Setting that allows Longhorn to generate ingress records. |
+| ingress.host | string | `"sslip.io"` | Hostname of the Layer 7 load balancer. |
+| ingress.ingressClassName | string | `nil` | IngressClass resource that contains ingress configuration, including the name of the Ingress controller. ingressClassName can replace the kubernetes.io/ingress.class annotation used in earlier Kubernetes releases. |
+| ingress.path | string | `"/"` | Default ingress path. You can access the Longhorn UI by following the full ingress path {{host}}+{{path}}. |
+| ingress.secrets | string | `nil` | Secret that contains a TLS private key and certificate. Use Secrets if you want to use your own certificates to secure Ingresses. |
+| ingress.secureBackends | bool | `false` | Setting that allows you to enable connections to the backend service via port 443. |
```

Comment:
`Setting that allows you to enable secure connections to the Longhorn UI service via port 443.`